packages/client/src/components/MovieCard/handlerFavorite.js

```diff
@@ -0,0 +1,57 @@
+import { apiURL } from '../../apiURL';
+
+export const handleAddFavorite = async (
+  movieID,
```

Comment:
MInor observation - Here is movieID and in below operation its movieId. It could be consistent

Reply:
I fixed the typo